components/core/tools/scripts/lib_install/manylinux_2_28/install-prebuilt-packages.sh

```
@@ -0,0 +1,68 @@
+# necessary to resolve build issues with third-party libraries. Even after
+# we added the "-DCMAKE_POLICY_VERSION_MINIMUM=3.5" flag, compilation issues
+# in libraries such as googletest-src and yaml-cpp still persist.
+# See sample error below:
+# CMake Error at _deps/googletest-src/CMakeLists.txt:4 (cmake_minimum_required):
+#   Support for CMake versions below 3.5 has been dropped.
+#
+#   Adjust the VERSION argument’s minimum value, or use the <min>...<max> syntax
+#   to indicate the required CMake version range.
+#
+#   Alternatively, you can try adding -DCMAKE_POLICY_VERSION_MINIMUM=3.5
+export PLATFORM=$(uname -m)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)